packages/core/src/services/chains/ChainValidator/index.ts

```diff
@@ -198,6 +205,17 @@ export const validateChain = async (
     config,
   })
 
+  const schema = getInputSchema({
+    config,
+    configOverrides,
+    ignoreSchema: promptlVersion === 0 && !chainCompleted,
+  })
+  const output = getOutputType({
+    config,
+    configOverrides,
+    ignoreSchema: promptlVersion === 0 && !chainCompleted,
+  })
```

Comment:
What happen with non-promptl prompts?

Reply:
old logic. Ignore schema in all steps but the last one